readme.md

```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://afesh4ak.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://afesh4ak.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Consider adding these links to the PR description

Reply:
I need help with icons,  I've tried many different methods, but the icons aren't behaving as I need them to.